zeton/templates/top.html

```diff
@@ -0,0 +1,18 @@
+{% block top %}
+            <section class="welcome">
+                <div class="col col-sm-12 col-md-12 col-lg-12">
```

Comment:
`col-sm-12` automatically makes every column that is bigger 12. `sm` automatically makes it `md` `lg` `xl`